Sources/Plasma/FeatureLib/pfPython/cyMiscGlue4.cpp

```diff
@@ -405,34 +405,50 @@ PYTHON_GLOBAL_METHOD_DEFINITION(PtDebugAssert, args, "Params: cond, msg\nDebug o
 PYTHON_GLOBAL_METHOD_DEFINITION_WKEY(PtDebugPrint, args, kwargs, "Params: *msgs, **kwargs\nPrints msgs to the Python log given the message's level")
 {
     uint32_t  level = cyMisc::kErrorLevel;
+    ST::string sep = ST_LITERAL(" ");
 
     do {
         // Grabbin' levelz
         if (kwargs && PyDict_Check(kwargs)) {
-            PyObject* value = PyDict_GetItem(kwargs, PyString_FromString("level"));
+            PyObject* value = PyDict_GetItemString(kwargs, "level");
             if (value) {
                 if (PyInt_Check(value))
                     level = PyInt_AsLong(value);
                 else
                     break;
             }
+
+            value = PyDict_GetItemString(kwargs, "sep");
```

Comment:
Unfortunately, `plStatusLog` unconditionally treats each call to `AddLine` as at least one line. Fixing that is outside the scope of this PR, but I will add an `end` argument that allows terminating with an arbitrary string.

Reply:
Fair enough, but I'd rather not see `end` default to `'\n'` if we're going to end up writing two newlines (one from `AddLine` and one from the default `end`)...  Maybe it would be better to just leave out the `end` parameter until it's needed (and then update the `DebugPrint` to use it properly at the same time)

Reply:
`plStatusLog` only adds the trailing `\n` if one is not present. I decided to default `end` to `\n` on the off chance that some later work allows eliding newlines in `plStatusLog` :smile: